README.md

```diff
@@ -1215,9 +1215,50 @@ Use the following setting to optimize the content share audio for an audio strea
 meetingSession.audioVideo.setContentAudioProfile(AudioProfile.fullbandMusicStereo());
 ```
 
+**Use case 35.** Redundant Audio
+
+Starting from version 3.17.0, the SDK starts sending redundant audio data to our servers on detecting packet loss
+to help reduce its effect on audio quality. Redundant audio packets are only sent out for packets containing active
+audio, ie, speech or music. This may increase the bandwidth consumed by audio to up to 3 times the normal amount
+depending on the amount of packet loss detected. The SDK will automatically stop sending redundant data if it hasn't
+detected any packet loss for 5 minutes.
+
+This feature is not supported on Firefox at the moment.
```

Comment:
So it works for Chrome, Safari, and iOS and Android?

Reply:
Yeah. Basically chromium and webkit browsers

Reply:
Technically we also support Opera. So maybe we should be more specific. Also will all browser version works or only a certain version?

Reply:
I added some extra information. In the latest push, I also added fallback to peer connection without RED if we determine that we cannot enable it. Tested this on an older version of safari on saucelabs and was able to send/receive audio/video fine.